docs/dnx/overview.rst

```diff
@@ -80,18 +82,16 @@ Dependencies in DNX consist of a name and a version number. Version numbers shou
     }
 
 
-The global.json file also defines the minimum DNX version ("sdk" version) needed to build the project.
+The *global.json* file also defines the minimum DNX version ("sdk" version) needed to build the project.
 
 Dependencies are transitive in that you only need to specify your top level dependencies. DNX will handle resolving the entire dependency graph for you using the installed NuGet packages. Project references are resolved at runtime by building the referenced project in memory. This means you have the full flexibility to deploy your DNX application as package binaries or as source code.
 
 .. _dnx-concept-packages-and-feeds:
 
 Packages and feeds
-------------------
-
-For package dependencies to resolve they must first be installed. You can use DNU to install a new package into an existing project or to restore all package dependencies for an existing project, like this::
+---------------------------
 
-.. code-block:: console
+For package dependencies to resolve they must first be installed. You can use DNU to install a new package into an existing project or to restore all package dependencies for an existing project. The following command downloads and installs all packages that are listed in the *project.json* file::
```

Comment:
Kind of an interesting distinction between "install" and "restore". I think the last sentence ought to say "The following ... and *RESTORES* all packages..." because it's *restoring*, not *installing*.